test/correctness/setup.py

```diff
@@ -35,8 +35,8 @@
         'pytest',
         'pymongo==3.6.1',
         'python-dateutil',
-        'PyYAML==3.12',
+        'PyYAML==4.2b4',
```

Comment:
Is there a reason why we not jut using the latest PyYAML?

Reply:
Its usually good idea to pin versions. We should do that for other packages as well.